internal/command/jsonformat/plan.go

```diff
@@ -454,7 +462,20 @@ func resourceChangeComment(resource jsonplan.ResourceChange, action plans.Action
 			buf.WriteString(fmt.Sprintf("\n  # (because key [%s] is not in for_each map)", resource.Index))
 		}
 		if len(resource.Deposed) != 0 {
-			// Some extra context about this unusual situation.
+			// In the case where we partially failed to replace a resource
+			// configured with 'create_before_destroy' in a previous apply and
+			// the deposed instance is still in the state, we give some extra
+			// context about this unusual situation.
+			buf.WriteString("\n  # (left over from a partially-failed replacement of this instance)")
+		}
+	case plans.Forget:
+		buf.WriteString(fmt.Sprintf("[bold]  # %s[reset] will be removed from the OpenTofu state [bold][red]but will not be destroyed[reset]", dispAddr))
```

Comment:
Nit: I know different people have thoughts on case fallthrough in go, but that might be a easy way to combine a lot of these two blocks?

Happy to merge without this!

Reply:
Please, please, don't :P

I'm worried someone will add a case not noticing the fallthrough and wreak havoc.

Reply:
It might reduce a few lines, but I agree with @cube2222 on this one. Plus, I think it's clearer when it's more explicit.